recipe/meta.yaml

```diff
@@ -1,25 +1,30 @@
-{% set version = "2.0" %}
+{% set version = "3.1.1" %}
 
 package:
   name: python-gflags
   version: {{ version }}
 
 source:
   fn: python-gflags-{{ version }}.tar.gz
-  url: https://pypi.python.org/packages/source/p/python-gflags/python-gflags-{{ version }}.tar.gz
-  md5: 23c9a793959a54971b1f094b0c6d03b1
+  # ugly url, anything better?
+  url: https://pypi.python.org/packages/ea/30/b8469c0d1837ce58fe3706e1f7169cbf6ca1fb87d1f84cece5182b67cb0b/python-gflags-3.1.1.tar.gz
+  md5: e8e8c17833e4f30304893021f6004006
 
 build:
   number: 1
-  skip: True  # [py3k]
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script:
+    # Does not work anymore
+    # - python setup.py install --single-version-externally-managed --record record.txt
+    - python setup.py install
+    - rm -Rf $PREFIX/*.egg-info
```

Comment:
Why doesn't `--single-version-externally-managed --record record.txt` work? Also why do we need to remove an `.egg-info` directory?